static/js/server_interaction.js

```diff
@@ -6,16 +6,14 @@
 
     //Connection information
     edx_adapt.apistr    = '/api/v1';
-    edx_adapt.server     = '52.210.53.36'
+    edx_adapt.server    = '52.210.53.36';
     edx_adapt.apiport   = '8080';
 
-    course_id = $(".xmodule_display", window.parent.document).attr('data-course-id');
+    edx_adapt.course_id = $(".xmodule_display", window.parent.document).data('course-id');
```

Comment:
Yesterday I've discovered that open edx relies on dom attributes rather than on in-memory, so it's better to use `.attr('data-...')` notation.
Please revert to be on a safe side.

NB https://coderwall.com/p/t_cgwq/when-is-better-to-use-data-or-attr